src/server/config.rs

```diff
@@ -83,7 +83,6 @@ pub struct Config {
     #[online_config(skip)]
     pub status_thread_pool_size: usize,
 
-    #[online_config(skip)]
```

Comment:
I don't see the point of changing this value online. It's only for controlling the size of a batch.

Reply:
The use case is from tidb-cloud to minimize the memory usage in a minimal test cluster. This kind of scenario is not common. I think it is harmless to support it.

Reply:
The batch is about 10MiB, and it only consumes about 10MiB * grpc_concurrency = 40MiB.